plugins/org.jkiss.dbeaver.tasks.ui.view/plugin.xml

```diff
@@ -27,6 +27,12 @@
             namespace="org.jkiss.dbeaver.task"
             properties="scheduled,editable"
             type="org.jkiss.dbeaver.model.task.DBTTask"/>
+        <propertyTester
```

Comment:
I don't like property testes which accept any objects in input. They produce a mess inside Eclipse registries.
Also maybe we should have property tested in ee only?